api/handlers.go

```diff
@@ -410,9 +410,9 @@ func postImagesLoad(c *context, w http.ResponseWriter, r *http.Request) {
 	wf := NewWriteFlusher(w)
 	callback := func(what, status string) {
 		if status == "" {
```

Comment:
Here we have to use string(Not a json):
fmt.Fprintf(wf, "%s:Loading Image... %s", what, status)